libs/perl.json

```diff
@@ -6,13 +6,15 @@
   {"id":"perl.YAML","imports":["YAML"],"name":"YAML","repo":"ingydotnet/yaml-pm","tags":["yaml"],"tech":["app-data"],"status":"model-ready"},
   {"id":"perl.xml-libxml","imports":["XML::LibXML"],"name":"XML-LibXML","repo":"shlomif/perl-XML-LibXML","tags":["xml"],"tech":["app-data"],"status":"model-ready"},
   {"id":"perl.rex","imports":["Rex"],"name":"Rex","repo":"RexOps/Rex","tags":["server-orchestration"],"tech":["cloud"],"status":"model-ready"},
-  {"id":"perl.ppi","imports":["PPI"],"name":"PPI","repo":"adamkennedy/PPI","tags":["perl-parsing"],"tech":["parsing"],"status":"awaiting-model"},
+  {"id":"perl.test-minimumversion","imports":["Test::MinimumVersion"],"name":"Test::MinimumVersion","repo":"karenetheridge/Dist-Zilla-Plugin-Test-MinimumVersion","tags":["min-versions"],"tech":["configuration"],"status":"awaiting-model"},
```

Comment:
Maybe, name == Test::Version ?

Because below, you use :: instead of '-'

Reply:
using '-' for consistency, so Test-MinimumVersion